src/runtime_src/core/tools/common/tests/TestValidateUtilities.cpp

```diff
@@ -137,39 +137,26 @@ get_instr_size(const std::string& dpu_file) {
 
 void
 wait_for_max_clock(int& ipu_hclock, std::shared_ptr<xrt_core::device> dev) {
```

Comment:
You don't need to pass `ipu_hclock` in the function

Reply:
I have changed the API to a returning type